scripts/cc-cli/src/commands/distributeRewards.ts

```diff
@@ -22,8 +20,6 @@ export function makeDistributeRewardsCommand() {
 async function distributeRewardsAction(options: OptionValues) {
   const { api } = await newApi(options.url);
 
-  const signerSeed = getSeedFromOptions(options);
-
   if (!options.validatorId) {
```

Comment:
Unrelated but what is my validator ID? How do I find it? Is it the validator address? 

Polkadot JS Explorer app has a "Pay All" button. I'll open a separate ticket for that.

Reply:
The Pay All button only applies to Validators you are nominating. It uses that list for the batch.